apiv2/oauth2_urls.py

```diff
@@ -46,6 +46,8 @@ def _wrapped_view_func(request, *args, **kwargs):
     return _wrapped_view_func
 
 
+app_name = 'oauth2_provider'
```

Comment:
Why is this required?

Reply:
This was to fix one of the deprecation warnings to update to 2.x. When URLs are included using `include`, the included URLs file now requires the `app_name` to be present. See some info: https://docs.djangoproject.com/en/4.1/topics/http/urls/#url-namespaces-and-included-urlconfs